site/en/docs/privacy-sandbox/videos/index.md

```diff
@@ -0,0 +1,320 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Privacy Sandbox video library'
+subhead: >
+  Find overviews, walkthroughs, and conceptual discussions in these Privacy Sandbox videos.
+description: >
+  Privacy Sandbox video library
+date: 2023-06-26
+authors:
+  - nmichell
+---
+
+If you're looking for Privacy Sandbox resources, check out the videos below. Some are recorded Privacy Sandbox office hours. You can find upcoming office hours on the [events page](/docs/privacy-sandbox/events/).
```

Comment:
I think the opener here should be more along the lines of describing the selection of videos, as in from past events, series, and office hours. We can probably assume that if the reader is on this page, then they are definitely looking for Privacy Sandbox resources. 😁

Reply:
PTAL @rowan-m